### DISCUSSION TITLE
Idea: Align the indent size of line number of `consult-ripgrep`?

### DISCUSSION THREAD

**Discussion by eval-exec** [minad/consult#912] [Q&A] [answered]
Hello,

![Screenshot_20231227_165436](https://github.com/minad/consult/assets/46400566/80f5beac-0b35-4b04-ad54-7e634a442e28)

I'm sure the line number in `consult-line`'s result is aligned well,
but the line number in `consult-ripgrep`'s result is not aligned well.

Is it eazy to make the line number in `consult-ripgrep` aligned well?

**Comment by minad** [comment, marked as answer]
The line numbers come from the grep/rg output and they are not aligned there. I am not sure if we should reformat the output in Consult.

**Reply by eval-exec** [reply]
Maybe we can provide an customizable option to enable/disable reformat the line number in `consult-ripgrep`' s output ?

**Reply by minad** [reply]
Yes, one can always add options. In this case I don't think we should. The format also has implications beyond Consult. It will affect embark-export, grep-mode, wgrep-mode, ... Maybe all of these already handle padded line numbers?

**Reply by eval-exec** [reply]
I agree with you.
